apps/site/src/app/(home)/sections/Involved/Involved.module.scss

```diff
@@ -0,0 +1,125 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as bootstrap;
+
+$primary-color: #78638A;
+$secondary-color: #DAFFF5;
+$white: #fff;
+$black: #000;
```

Comment:
Amazing work! The text color and border color aren't exactly #000, so please refer to the figma for accurate colors. Also, please use the colors found in _zothacks-theme.scss if possible. Otherwise, you can the colors declared here